cap/modules/repos/gitlab_api.py

```diff
@@ -159,3 +152,27 @@ def _get_branch_and_sha(self, branch_or_sha):
                         f'{branch_or_sha} does not match any branch or sha.')
 
         return branch, sha
+
+    @classmethod
+    def create_repo(cls, user_id, repo_name, host):
+        token = cls._get_token(user_id)
+        if not token:
+            raise GitUnauthorizedRequest(
+                'Gitlab requires authorization - '
+                'connect your CERN account: Settings -> Integrations.')
+
+        try:
+            gitlab = Gitlab(f'https://{host}', oauth_token=token)
```

Comment:
Need to check that `host` is correct.. If someone provides Gitlab link from official gitlab this will fail on the token fetch

Reply:
Fixed in #2441